libs/application/templates/login-service/src/fields/InformationAboutApplication/InformationAboutApplication.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useEffect, useState } from 'react'
+import {
+  FieldBaseProps,
+  formatText,
+  getErrorViaPath,
+  getValueViaPath,
+} from '@island.is/application/core'
+import { GridColumn, GridRow, AlertMessage } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import {
+  CompanySearchController,
+  InputController,
+} from '@island.is/shared/form-fields'
+import { applicant } from '../../lib/messages'
+import { gql, useLazyQuery } from '@apollo/client'
+import { useFormContext } from 'react-hook-form'
+import { RskCompany } from '@island.is/api/schema'
+
+export const COMPANY_REGISTRY_COMPANY = gql`
+  query GetCompany($input: RskCompanyInfoInput!) {
+    companyRegistryCompany(input: $input) {
+      name
+      nationalId
+      companyInfo {
+        vat {
+          dateOfRegistration
+          dateOfDeregistration
+          classification {
+            type
+            classificationSystem
+            number
+            name
+          }
+        }
+      }
+    }
+  }
+`
+
+interface CompanyRegistryCompany {
+  companyRegistryCompany: RskCompany
+}
+
+interface SearchField {
+  nationalId: string
+  label: string
+}
+
+export const InformationAboutApplication = ({
+  application,
+  errors,
+}: FieldBaseProps) => {
+  const { setValue } = useFormContext()
+  const { formatMessage } = useLocale()
+  const [nationalId, setNationalId] = useState<string>('')
+  const [isatNumber, setIsatNumber] = useState<string>('')
```

Comment:
You can use `watch` from `react-hook-form` to track the actual value instead of storing it in two places :)

Reply:
Cool, I'll look into that! 😄